app/check_dataset.py

```diff
@@ -187,15 +206,39 @@ def check_history(ds: DataSource) -> ResultLog:
 
     log = ResultLog()
 
+    exceptions_cached = pd.read_csv('./resources/exceptions.csv', parse_dates=['date'])
```

Comment:
Is this the right place for exceptions? Maybe we have an anomalies directory and then we track these by type (state, us, etc).

Reply:
nah, because it is under the source tree and we don't want it in revision control (if think).

Can you add a log dir to the .ini file, create the dir if it is missing, and default it to ~/logs/exceptions.csv?

Reply:
I was thinking we _would_ want it under source control. This way, we can detect when a new anomaly is detected and we can alert the channel. If an anomaly is valid, we simply add it to the cached file so the room never gets alerted on that anomaly again. These would be fairly small files -- the current exceptions has 20 exceptions and is < 1KB is size.